### DISCUSSION TITLE
Cloning Existing Aircraft

### DISCUSSION THREAD

**Discussion by bostonmyk** [PygmalionOfCyprus/cmo-db-requests#30] [Suggestions]
Hi guys. The Aircraft form requires dimension fields and things like that but that is redundant for types that already exist in the db. Any chance you can make those not required. I know why you're doing this for new entries but for old, you're making work for us for no real outcome.

Thanks!

Mike

**Comment by PygmalionOfCyprus** [comment]
Good point!

Would it be helpful for me to add a second aircraft form ("Aircraft Copy" or something) with certain fields no longer marked required (or even removed altogether)? We could extend this to Facilities, Ships, and Submarines as well.

We were actually considering segmenting the "addition" label into "addition" and "copy" (or something), so this would fit nicely with that.

**Comment by bostonmyk** [comment]
Sure or could add a..is this a copy of another nation's aircraft field or something like that that asks for the id number. Likewise same for copyover from the CWDB. So maybe

If this unit can be copied over from a valid existing entry please provide DB to copy from and ID number. Followed by a dropdown for the DB and text field for ID.

Mike

**Comment by PygmalionOfCyprus** [comment]
Sorry it took so long to act on this. There is now a new "Copy Entry" form and corresponding label. I'm also tweaking the other forms to allow people to suggest "bases" from which to work off of (such as "new" platforms based on others); in those cases they'll still need to provide required info but they can more easily reference another DBID.

**Reply by bostonmyk** [reply]
Awesome man. Good job with all this!

Mike
